scripts/e2e-build-package-publish.ts

```diff
@@ -1,6 +1,10 @@
 import { execSync } from 'child_process';
-import { readFileSync, removeSync, writeFileSync } from 'fs-extra';
-import { readdirSync } from 'fs';
+import { removeSync } from 'fs-extra';
+import { readFileSync, writeFileSync, readdirSync } from 'fs';
+import {
+  prettierVersion,
+  typescriptVersion,
+} from '../packages/workspace/src/utils/versions';
```

Comment:
@meeroslav The TypeScript and Prettier versions have not been dropped they get there values from the versions used in @nrwl/workspace